Logging/Log.cs

```diff
@@ -140,6 +140,25 @@ public static void Error(string format, params object[] args)
         }
 
         /// <summary>
+        /// Log trace with format string
+        /// </summary>
+        public static void TraceFormat(string traceFormat, params object[] args)
```

Comment:
I just noticed the params overload was added two weeks ago, in the Toolbox import. :)
I'll just remove TraceFormat and use the new overload.